theme.json

```diff
@@ -1051,6 +1051,58 @@
 					"fontSize": "var:preset|font-size|x-large"
 				}
 			},
+			"core/quote": {
+				"border": {
```

Comment:
I believe these should not go into the quote core styles, as they're different. Please check the Figma designs for the quote [here](https://www.figma.com/design/dzGCSntVch4EQdVERTqyVK/Twenty-Twenty-Five?node-id=77-1417&t=iVc8k2Kl9C1qrITw-1).

Reply:
Thanks I didn't see that part of the figma. Have updated this and added a new comment below https://github.com/WordPress/twentytwentyfive/pull/118#issuecomment-2322611124